### ISSUE TITLE
Replace envelope icon

### ISSUE THREAD

**Issue by licaon-kter** [yaxim-org/yaxim#266]
...with something not-email related, in chat view.

That's the message received checkmark right?

**Comment by ge0rg** [comment]
Yes, it's the message-delivered checkmark, and that icon also conveys the following other states:

- message stored locally for sending
- message sent but not acked
- message rejected
- message delivered
- TODO: message read by recipient
- TODO: message sent to server but not yet XEP-0198 acknowledged

If you are willing to provide SVG images for all of the above (all images need to have the same width, and they need to work on dark and on bright background), I'm not opposed to getting rid of the envelopes.
